platform/android/java/app/build.gradle

```diff
@@ -12,6 +12,7 @@ allprojects {
         mavenCentral()
         gradlePluginPortal()
         maven { url "https://plugins.gradle.org/m2/" }
+        maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/"}
```

Comment:
Just curious about what this change is for.

Reply:
This provides access to the snapshots repos, which we use for non-stable releases.